css/style.css

```diff
@@ -323,3 +271,68 @@ th {
 	font-weight: bold;
 	border: none;
 }
+
+/* Checkbox css */
```

Comment:
Any particular reason to move the checkbox CSS?

Reply:
I was trying to keep the css more consistent because I thought it was intentional to have the component type of css like a checkbox/tooltip at the bottom of the file. I can make a quick change and move it back if you prefer it the other way 😁

Reply:
That makes sense, no need to change it then. 😄

Reply:
Ahh, sorry for the trouble. 😄